civihr_employee_portal/views/includes/civihr_employee_portal_handler_aggregated_address.inc

```diff
@@ -0,0 +1,81 @@
+<?php
+
+use CRM_Utils_Array as ArrayHelper;
+
+/**
+ * Handles aggregation of a contacts different address fields
+ */
+class civihr_employee_portal_handler_aggregated_address extends views_handler_field {
+
+  /**
+   * @inheritdoc
+   */
+  public function render($values) {
+    $contactID = isset($values->id) ? $values->id : NULL;
+
+    if (!$contactID) {
+      return '';
+    }
+
+    $address = $this->getAddress($contactID);
+
+    if (!$address) {
+      return '';
+    }
+
+    return $this->formatAddress($address);
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public function query() {
+    // override default behaviour as this handler is not tied to a single field
+  }
+
+  /**
+   * Gets the primary address for a contact ID
+   *
+   * @param int $contactID
+   * @return array|null
+   */
+  private function getAddress($contactID) {
+    $return = [
+      'street_address',
+      'supplemental_address_1',
+      'city',
+      'state_province_id.name',
+      'postal_code',
+      'country_id.name',
+    ];
+
+    $params = ['is_primary' => 1, 'contact_id' => $contactID, 'return' => $return];
+    $address = civicrm_api3('Address', 'get', $params);
+
+    if ($address['count'] != 1) {
+      return NULL;
+    }
+
+    return array_shift($address['values']);
+  }
+
+  /**
+   * @param array $address
+   *   The address data
+   * @return string
+   *   The formatted address
+   */
+  private function formatAddress($address) {
+    $format = '%s<br/>%s<br/>%s %s<br/>%s<br />%s';
```

Comment:
Seems the space before the closing tag for the last `<br />` is not consistent with the others.
probably make all to have a space before the closing tag.